test/pgo_basic_SUITE.erl

```diff
@@ -204,3 +205,62 @@ types(_Config) ->
     #{command := select, rows := [Row17, Row18]} = R3,
     ?assertMatch({?TXT_UUID}, Row17),
     ?assertMatch({?TXT_UUID}, Row18).
+
+numerics(_Config) ->
+    %% TODO:
+    %% - Test non-param based query (i.e., select 1234:numeric(0, 1) )
+    %% - Test precision and scale query params
+    %% - Test tuple param which allows you to pass in your own dweight and dscale
+
+     BasicQuery = "select $1::numeric",
+    ?assertMatch(#{rows := [{1}]}, pgo:query(BasicQuery, [1])),
+    ?assertMatch(#{rows := [{-1}]}, pgo:query(BasicQuery, [-1])),
+    ?assertMatch(#{rows := [{1.1}]}, pgo:query(BasicQuery, [1.1])),
+    ?assertMatch(#{rows := [{-1.1}]}, pgo:query(BasicQuery, [-1.1])),
+    ?assertMatch(#{rows := [{1.12345}]}, pgo:query(BasicQuery, [1.12345])),
+    ?assertMatch(#{rows := [{1.0e-9}]}, pgo:query(BasicQuery, [0.000000001])),
+    ?assertMatch(#{rows := [{-1.0e-9}]}, pgo:query(BasicQuery, [-0.000000001])),
+    ?assertMatch(#{rows := [{1.0e-10}]}, pgo:query(BasicQuery, [0.0000000001])),
+    ?assertMatch(#{rows := [{-1.0e-10}]}, pgo:query(BasicQuery, [-0.0000000001])),
+    ?assertMatch(#{rows := [{1.0e-11}]}, pgo:query(BasicQuery, [0.00000000001])),
+    ?assertMatch(#{rows := [{-1.0e-11}]}, pgo:query(BasicQuery, [-0.00000000001])),
+    ?assertMatch(#{rows := [{1.0e-32}]}, pgo:query(BasicQuery, [1.0e-32])),
+
+
+     #{command := create} = pgo:query("create table numeric_tmp (id integer primary key, a_int integer, a_num numeric)"),
```

Comment:
Can you add a column that actually sets the weight/scale, like `b_num numeric(5,3)` or whatever and insert/select from it too.

Reply:
Done